pyne/material.pyx

```diff
@@ -1234,6 +1234,51 @@ cdef class _Material:
 
         self.mat_pointer.from_atom_frac(af)
 
+    def from_activity(self, activities):
+        """from_activity(activities)
+        Loads the material composition based on a mapping of radionuclide
+        activities.
+
+        Parameters
+        ----------
+        activities : dict
+            Dictionary that maps radionuclides to activities for the material.
+            The keys may be intergers or strings. The values must be castable
+            to floats.
```

Comment:
I think we need to describe the assumed units of the activity (Bq) here.

It only matters if the user is relying on the calculation of the total mass to be correct - in this method, the mass is always determined by the `norm_comp()` method.